pyvista/core/dataset.py

```diff
@@ -1094,13 +1526,26 @@ def __str__(self) -> str:
         return self.head(display=False, html=False)
 
     def overwrite(self, mesh: _vtk.vtkDataSet):
-        """Overwrite this mesh inplace with the new mesh's geometries and data.
+        """Overwrite this dataset inplace with the new dataset's geometries and data.
 
         Parameters
         ----------
         mesh : vtk.vtkDataSet
             The overwriting mesh.
 
+        Examples
+        --------
+        Create two meshes and overwrite ``mesh_a`` with ``mesh_b``.
+        Show that the array of points from ``mesh_a`` are identical and point
+        to the points in ``mesh_b``.
+
+        >>> import pyvista
+        >>> mesh_a = pyvista.Sphere()
+        >>> mesh_b = pyvista.Cube().clean()
+        >>> mesh_a.overwrite(mesh_b)
+        >>> id(mesh_a.points) == id(mesh_b.points)
```

Comment:
`id` is usually a red herring, and it can easily confuse new users. I recommend using `is` instead.
```suggestion
        >>> mesh_a.points is mesh_b.points
```